middlewares/auth/usersTokenAuth.middleware.js

```diff
@@ -0,0 +1,42 @@
+const jwt = require("jsonwebtoken");
+const { endpointResponse } = require("../../helpers/success");
+const { ErrorObject } = require("../../helpers/error");
+require("dotenv").config();
+
+module.exports = async (req, res, next) => {
+    
+  //Get Token from Headears
+  const token = req.headers.authorization.split(" ")[1];
+
+  try {
+
+    //Check thath token exist
+    if (!token) {
+      endpointResponse({
+        res,
+        code: 404,
+        message: "missing token",
+      });
+    } else {
+
+      //Check token validation
+      await jwt.verify(token, process.env.SECRET, (err, decoded) => {
```

Comment:
implement the method already created:

![Captura de Pantalla 2022-11-12 a la(s) 1 51 28 p m](https://user-images.githubusercontent.com/84211273/201490087-f6261d9e-ccc5-4cb5-9fde-cbae3deca0a3.png)